src/components/player-controls.ts

```diff
@@ -45,6 +55,13 @@ class PlayerControls extends LitElement {
             ></mxmp-volume>
           `,
         )}
+        ${when(
+          ['off'].includes(this.activePlayer.state),
```

Comment:
Why don't you want to show the power button if it is on?

Reply:
The power button is already shown in the volume control area by the upper WHEN statement if the player is PAUSED or PLAYING:
        ${when(
          ['paused', 'playing'].includes(this.activePlayer.state),
          ...
            <mxmp-volume
              .store=${this.store}
              .player=${this.activePlayer}
              .updateMembers=${!this.config.playerVolumeOnlyAffectsMainPlayer}
            ></mxmp-volume>